etc/.release-please-manifest.json

```diff
@@ -6,5 +6,6 @@
   "charts/cu-up": "8.0.2",
   "charts/drax": "8.0.2",
   "charts/du-metrics-server": "0.2.1",
-  "charts/xapp-hello-world": "3.0.1"
+  "charts/xapp-hello-world": "3.0.1",
+  "charts/telemetry-collector": "1.0.0"
```

Comment:
In `Chart.yaml` the initial version was set to 0.9.0. This should match for the initial publishing.

Reply:
Changed to 1.0.0